ee_plugin/processing/export_geotiff.py

```diff
@@ -22,11 +22,17 @@ class ExportGeoTIFFAlgorithm(QgsProcessingAlgorithm):
     OUTPUT = "OUTPUT"
 
     def initAlgorithm(self, config: dict) -> None:
-        raster_layers = [
-            layer.name()
-            for layer in Map.get_iface().mapCanvas().layers()
-            if layer.providerType() == "EE"
-        ]
+        try:
+            raster_layers = [
+                layer.name()
+                for layer in Map.get_iface().mapCanvas().layers()
+                if layer.providerType() == "EE"
+            ]
+        except Exception as e:
```

Comment:
Did you actually experience errors here?  This seems like a pretty benign operation.